README.md

```diff
@@ -25,3 +25,112 @@ Swagger configuration params that can be set in microfocus-config.js:
  - deepLinking: true
 
 For a complete list of swagger configuration params refer [Swagger configuration params](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md#core).
+
+## Using this module in Spring Boot
+
+[Here](https://github.com/CAFapi/caf-swagger) is an example of how this swagger-ui can be used in a Spring Boot application : 
+1. Add the module as a `runtime` dependency.
```

Comment:
Which module are you referring to now?  This one or the `com.github.cafapi:swagger-ui-annotation` one?

Reply:
This one

Reply:
Is that necessary?  I don't see it in [this](https://github.com/CAFDataProcessing/staging-service/commit/f4a051f08b522a1cf806083dd1d1b2237a88cddc) commit.